src/custom-types.json

```diff
@@ -93,6 +92,9 @@
   "application/msword": {
     "compressible": false
   },
+  "application/node": {
+    "extensions": ["cjs"]
```

Comment:
Please add a primary source link for this data.

Reply:
@jkrems ?

Reply:
The primary source should be https://www.iana.org/assignments/media-types/application/node, but this needs updating to add `.cjs`.

Node’s definition of `.cjs` is here: https://nodejs.org/api/esm.html#esm_enabling

Reply:
Update: https://www.iana.org/assignments/media-types/application/node now includes `.cjs`.